packages/typo3-docs-theme/src/Inventory/DefaultInventories.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace T3Docs\Typo3DocsTheme\Inventory;
+
+enum DefaultInventories: string
+{
+    // Core Manuals
+    case t3coreapi = 'https://docs.typo3.org/m/typo3/reference-coreapi/{typo3_version}/en-us/';
+    case t3tca = 'https://docs.typo3.org/m/typo3/reference-tca/{typo3_version}/en-us/';
+    case t3tsconfig = 'https://docs.typo3.org/m/typo3/reference-tsconfig/{typo3_version}/en-us/';
+    case t3tsref = 'https://docs.typo3.org/m/typo3/reference-typoscript/{typo3_version}/en-us/';
+    case t3viewhelper = 'https://docs.typo3.org/other/typo3/view-helper-reference/{typo3_version}/en-us/';
+
+    // Official Core Tutorials and Guides
+    case t3editors = 'https://docs.typo3.org/m/typo3/tutorial-editors/{typo3_version}/en-us/';
+    case t3install = 'https://docs.typo3.org/m/typo3/guide-installation/{typo3_version}/en-us/';
+    case t3sitepackage = 'https://docs.typo3.org/m/typo3/tutorial-sitepackage/{typo3_version}/en-us/';
+    case t3start = 'https://docs.typo3.org/m/typo3/tutorial-getting-started/{typo3_version}/en-us/';
+    case t3translate = 'https://docs.typo3.org/m/typo3/guide-frontendlocalization/{typo3_version}/en-us/';
+    case t3ts45 = 'https://docs.typo3.org/m/typo3/tutorial-typoscript-in-45-minutes/{typo3_version}/en-us/';
+
+    // Team Guides, they are commonly not versioned
+    case h2document = 'https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/';
+    case t3content = 'https://docs.typo3.org/m/typo3/guide-contentandmarketing/main/en-us/';
+    case t3contribute = 'https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/en-us/';
+
+    // System Extensions
+    case ext_adminpanel = 'https://docs.typo3.org/c/typo3/cms-adminpanel/{typo3_version}/en-us/';
+    case ext_core = 'https://docs.typo3.org/c/typo3/cms-core/{typo3_version}/en-us/';
```

Comment:
```suggestion
    case ext_core = 'https://docs.typo3.org/c/typo3/cms-core/main/en-us/';
```

Changelog is always "main".

Reply:
true - also as the changelog is really something else then just any system extension manual (and also handled differently with the version) I moved it to its own section.